torch/jit/__init__.py

```diff
@@ -767,7 +786,9 @@ def trace(func,
         original ``nn.Module``.
         If ``callable`` is a standalone function, ``trace`` returns ``torch._C.Function``
 
-    Example (tracing a function)::
+    Example (tracing a function):
```

Comment:
Why is it `::` -> `:` ?

Reply:
`::` is for a code block but now that's done by `testcode`